MauricioSight/landing-page-acaso
# feature: coincidence and see more scren and worker-loader

## What I did

- Coincidence and See More screen
- Generic animated paragraph component
- worker loader install

## How to test

- Run `yarn install` to install the new library worker-loader
- Scroll to see the screens
